lxqt-ts2desktop

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# to be executed in parent directory of /translations
+
+Component=${PWD##*/}
+PARENTDIR=$(basename $(dirname $PWD))
```

Comment:
I don't think, that driving the logic based on "current directory" is a good idea in long term.
Wouldn't it be better to give the script parameters? E.g.
- <component>
- <source_ts_translations_dir>
- <destination_desktop_translations_dir>

Then the `case` can be completely excluded (and no copy-paste...)

Reply:
The parameters would then be passed by cmake? If this is possible it is surely a much better approach, but Cmake syntax and powers are quite new for me.